packages/@aws-cdk/aws-appsync/test/integ.graphql.ts

```diff
@@ -71,7 +71,17 @@ const orderTable = new Table(stack, 'OrderTable', {
   },
   removalPolicy: RemovalPolicy.DESTROY,
 });
-const paymentTable =  Table.fromTableName(stack, 'PaymentTable', 'PaymentTable');
+
+new Table(stack, 'PaymentTable', {
+  billingMode: BillingMode.PAY_PER_REQUEST,
+  partitionKey: {
+    name: 'id',
+    type: AttributeType.STRING,
+  },
+  removalPolicy: RemovalPolicy.DESTROY,
+});
+
+const paymentTable =  Table.fromTableName(stack, 'ImportedPaymentTable', 'PaymentTable');
```

Comment:
good catch on this!
would be good to add stack verification steps: i.e. [stepfunctions integ test](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-stepfunctions-tasks/test/lambda/integ.invoke.ts#L6-L17)